gvsbuild/projects/adwaita_icon_theme.py

```diff
@@ -27,15 +27,14 @@ def __init__(self):
         Meson.__init__(
             self,
             "adwaita-icon-theme",
-            version="44.0",
+            version="45.beta",
```

Comment:
any reason to update to beta?

Reply:
The current patch we are carrying has an issue with generating the correct icons. The beta version includes all the upstream fixes. If you would prefer us not using this version, I can update our patch.

Reply:
I am usually a bit hesitant to update to beta versions.... although this is the icon theme and as you say the patches are upstream so I will let to you to decide if you tested it.

Reply:
Thanks. Yup, the only changes here are the meson build system and some small icon changes, so I think using the beta version will be fine.